Cabal/tests/PackageTests.hs

```diff
@@ -25,9 +27,12 @@ import Distribution.Verbosity (normal, flagToVerbosity)
 import Distribution.ReadE (readEOrFail)
 
 import Control.Exception
+import Data.Typeable (Proxy(..), Typeable)
```

Comment:
BTW, this broke the build on GHC <= 7.6. I changed this line to import `Data.Proxy` instead and added a dependency on `tagged`.